### PR TITLE
Adding schedule fields to NewAzMaintenance Configuration cmdlet and new Public Configuration GET cmdlet

### PR DESCRIPTION
**Pull request by sotaneja**
## Description

Adding new GET  cmdlet for new resource type public maintenance configurations
Adding optional schedule related fields for New-AzMaintenanceConfiguration cmdlet

Design Review: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/693

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by isra-fel** [comment]
```
Target                                  Description
  ------                                  -----------
  Get-AzMaintenancePublicConfiguration.md Online version in the header of the file is incorrect. The corresponding URL s.
 ```
Please fix the URL issue in docs

File: src/Maintenance/Maintenance/help/Get-AzMaintenancePublicConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+```
+
+## DESCRIPTION
+Get Public Maintenance Configuration record
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzMaintenancePublicConfiguration -Name workervmscentralus
```
Comment (isra-fel) [inline src/Maintenance/Maintenance/help/Get-AzMaintenancePublicConfiguration.md:27]: Are you missing `-ResourceGroupName`?

**Comment by isra-fel** [comment]
@sotaneja please never add any `## Version x.y.z` title in `ChangeLog.md`. Just put your change logs beneath `Upcoming Release`.

I updated it for you but please be careful about it next time.

THanks!

**Pull request merged**